.ci/README.md

```diff
@@ -0,0 +1,179 @@
+
+## How are the projects related between them?
+
+As we have commented above, we delegate this part to build-chain tool and the project depdency information is externalize in one single file. See [project-dependencies.yaml file](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/.ci/project-dependencies.yaml).
+
+There you will find a list of projects (all the projects involved on the different RHBA products, 7.x, 8.x ...) and the dependencies between them. For instance
+
+```
+  - project: kiegroup/lienzo-core
+
+  - project: kiegroup/lienzo-tests
+    dependencies:
+      - project: kiegroup/lienzo-core
+```
+
+means `kiegroup/lienzo-tests` project is part of the RHBA project set and depends on `kiegroup/lienzo-core` project (with no additional dependencies) which is also defined right on top of it.
+
+> **_Note:_** So in case `lienzo-tests` pull request is open a job checking out `lienzo-core` and `lienzo-tests` will be launched, building `lienzo-core` first and then `lienzo-tests`.
+
+This way you can easily understand the rest of document.
+
+### What about this mapping thing?
+
+You will be able to see more complex definitions for projects like `drools` or `optaplanner`. For instance:
+
+```
+ - project: kiegroup/drools
+    dependencies:
+      - project: kiegroup/kie-soup
+      - project: kiegroup/droolsjbpm-knowledge        
+    mapping:
+      dependencies:
+        default:
+          - source: 7.x
+            target: main
+      dependant:
+        default:
+          - source: main
+            target: 7.x
+      exclude:
+        - kiegroup/optaweb-employee-rostering
+        - kiegroup/optaweb-vehicle-routing
+        - kiegroup/optaplanner          
+        - kiegroup/droolsjbpm-knowledge  
+```
+
+`mapping` is about "branch mapping" and we require to take different branches per project depending on the project triggering the job (the project that launches opens the PR) and the target branch for a particular PR. In this case we want to get `drools:7.x` for every pull request targeting `main` branch, except for `kiegroup/opta*` and `kiegroup/droolsjbpm-knowledge`.
+We also want the opposite, to get `drools:main` for every pull request targeting `7.x`, the rest of branches will be a basic flat mapping from A to A always.
+
+## Productization nightly job
+
+The nightly jobs are totally different to the pull request jobs (but we are planning to move them to a closer approach).
+
+The job is a [multibranch pipeline](https://www.jenkins.io/doc/book/pipeline/multibranch/#creating-a-multibranch-pipeline) job getting the pipeline from [Jenkinsfile.nightly file](jenkins/Jenkinsfile.nightly). The high level process flow is:
+
+1. Reads project collection
+2. Reads Build Configuration
+3. Executes POM Manipulation Extension (PME)
+4. Executes Maven (the build itself)
+5. Uploads artifacts to the internal nexus repository
+6. Uploads properties to rcm-guest service
```

Comment:
```suggestion
6. Uploads properties to remote storage place for archive purpose.
```
I wold avoid to mention the service as it is a generally storage for later usage.